### ISSUE TITLE
Training error always fluctuates and doesn't decrease.

### ISSUE THREAD

**Issue by liuzhi136** [apache/incubator-mxnet#8341]
I had implemented a model which defined in "Training RNNs as Fast as CNNs". The structure I wrote as below:
![1](https://user-images.githubusercontent.com/13534043/31749188-fd014a78-b4aa-11e7-82e5-10732df9cb2a.png)
![2](https://user-images.githubusercontent.com/13534043/31749189-fd3d7e30-b4aa-11e7-95a2-89fc72078226.png)
![3](https://user-images.githubusercontent.com/13534043/31749190-fd72ba0a-b4aa-11e7-95cc-1145d1da7b5c.png)
The training and validation error looks like:
![screenshot from 2017-10-19 08-50-15](https://user-images.githubusercontent.com/13534043/31749202-149d4f9c-b4ab-11e7-8f80-7b7ea7a1cd0f.png)
The training data I used:
https://raw.githubusercontent.com/harvardnlp/sent-conv-torch/master/data/rt-polarity.all
I really don't know why this model fluctuates all the time. Does any has any idea for this?
I really need help to solve this immediately. Any help for this will be appreciated!

**Comment by liuzhi136** [comment]
@szha Do you have any idea about this?

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.
